edb/schema/objects.py

```diff
@@ -170,7 +205,7 @@ def __init__(self, type_, *, coerce=False,
         self.coerce = coerce
         self.allow_ddl_set = allow_ddl_set
 
-        self.compcoef = compcoef
+        self.compcoef = compcoef  # XXX what's up with it defaulting to None?
```

Comment:
`None` means "don't compare"

Reply:
Yes but just above you're declaring Field.compcoef to be strictly a float. I'll make it `Optional[float]` there.